jmeter/plans/apply.rb

```diff
@@ -0,0 +1,92 @@
+require 'ruby-jmeter'
+
+BASEURL = ENV.fetch('JMETER_TARGET_BASEURL')
+
+def url(path)
+  BASEURL + path
+end
+
+# 975 concurrent users, ~7 minutes per session
+test do
+  cookies clear_each_iteration: true
+  thread_count = 975
```

Comment:
Is a thread count this high workable? If not, do we have other options for concurrent execution of these steps?